generate-sln.ps1

```diff
@@ -0,0 +1,2 @@
+Remove-Item $PSScriptRoot/build -Recurse -Force -ErrorAction:SilentlyContinue -Confirm:$False | Out-Null
```

Comment:
Is this needed given MSVC supports CMake natively?

Reply:
I kept this line from legacy scripts, that if a rebuild of the project failed, the follow up attempt would not read the leftover CMakeCache.